BinaryClock/Display.cpp

```diff
@@ -46,17 +46,53 @@ void Display::print_static_text(const String &text)
     delay(1000);
 }
 
+ // void Display::print_time_binary(const Time &val)
+ // {
+ //     MATRIX.clear();
+ //
+ //     String digits = val.to_binary_string();
+ //     for (int i = 0; i < digits.length(); i++) {
+ //         if (digits.charAt(i) == '1') {
+ //             unsigned height = MATRIX.height();
+ //             MATRIX.drawFastVLine(i, 0, height, LED_ON);
+ //         }
+ //     }
+ //
+ //     MATRIX.writeDisplay();
+ // }
+
 void Display::print_time_binary(const Time &val)
 {
-    MATRIX.clear();
+   MATRIX.clear();
 
-    String digits = val.to_binary_string();
-    for (int i = 0; i < digits.length(); i++) {
-        if (digits.charAt(i) == '1') {
-            unsigned height = MATRIX.height();
-            MATRIX.drawFastVLine(i, 0, height, LED_ON);
-        }
-    }
+   String digits = val.to_binary_string();
+   for (int i = 0; i < 5; i++)
+   {
+       if (digits.charAt(i) == '1'){
+       unsigned height = MATRIX.height();
+       MATRIX.drawFastVLine(i, 0, height, LED_ON);
```

Comment:
I think that's fine. My "3" offset above assumed that the colon dots were 1 pixel wide, so adjust it as necessary so it looks correct on your end.

Given that the screen is 16px wide, and the digits are 11px wide + the 2px colon, this would mean the offset should be `(16 - 13) - 1 = 2`.